public/index.pug

```diff
@@ -29,22 +29,40 @@ html(lang="en-US")
         li(class="header__list-item")
-          a(href="https://unpkg.com/browse/simple-icons/") Unpkg
+          a(class="header__list-icon" title="Unpkg (CDN)" href="https://unpkg.com/browse/simple-icons/" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M 12.000002,-1.0439e-4 A 12.000002,12.000002 0 0 0 0,11.999898 12.000002,12.000002 0 0 0 12.000002,23.9999 12.000002,12.000002 0 0 0 24.000005,11.999898 12.000002,12.000002 0 0 0 12.000002,-1.0439e-4 Z m 0,2.00178449 A 10.008923,10.008923 0 0 1 22.008925,12.010603 10.008923,10.008923 0 0 1 12.000002,22.019525 10.008923,10.008923 0 0 1 1.9910798,12.010603 10.008923,10.008923 0 0 1 12.000002,2.0016801 Z m 2.140946,3.9286359 -0.03126,4.046388 -0.03329,4.046388 c -0.730063,2.108832 -3.414809,2.108832 -3.992864,0.007 C 10.008601,13.719677 9.9657825,12.23172 9.9657825,9.7482222 V 5.9373383 H 6.8400013 l 0.04314,4.1427307 c 0.053417,4.474577 0.08446,4.742195 0.6615523,5.909011 0.3532561,0.685103 1.2096345,1.445138 1.9696703,1.723461 1.3916151,0.524532 3.6824271,0.524532 5.0740421,0 0.749331,-0.278323 1.552186,-0.975201 1.937556,-1.680642 0.58876,-1.102587 0.654059,-1.627119 0.654059,-6.016059 l 0.0098,-4.0142733 -3.040143,-0.064657 z")
```

Comment:
Looks like this SVG could be optimized with a lower precisions 🤔

Reply:
true. done.